Tensile/Utils.py

```diff
@@ -48,7 +48,7 @@ class ProgressBar:
         createTime: The timestamp when the progress bar was created.
         message: The message displayed alongside the progress bar.
     """
-    def __init__(self, maxValue: int, desc: str, width=40):
+    def __init__(self, maxValue: int, desc: str, width=25):
```

Comment:
Was this supposed to be included in this PR?

Reply:
No, it was not.